app/assets/stylesheets/app/tags-show.scss

```diff
@@ -0,0 +1,28 @@
+.tag-container {
+  margin: 5px;
+  text-align: center;
+  text-decoration: none;
+  width: 180px;
+  font-size: 14px;
+  border-radius: 15px;
+  padding: 2px;
+  margin-bottom: 5px;
+  background-color: #a2918f;
+  color: rgb(255, 255, 255);
+  display: inline-block;
+
+  .tag-name {
+    display: inline;
+  }
+
+  .btn-tag.show {
+    float: right;
+    margin-right: 10px;
+    border-radius: 15px;
+    background-color: #a2918f;
+    color: rgb(255, 255, 255);
+    border-style: none;
```

Comment:
**[stylelint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected value "none" for property "border-style"   declaration-property-value-blacklist